utils/zerovec/src/varzerovec/databake.rs

```diff
@@ -2,10 +2,10 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-use crate::{ule::VarULE, VarZeroSlice, VarZeroVec};
+use crate::{ule::VarULE, varzerovec::VarZeroVecFormat, VarZeroSlice, VarZeroVec};
 use databake::*;
 
-impl<T: VarULE + ?Sized> Bake for VarZeroVec<'_, T> {
+impl<T: VarULE + ?Sized, F: VarZeroVecFormat> Bake for VarZeroVec<'_, T, F> {
     fn bake(&self, env: &CrateEnv) -> TokenStream {
         env.insert("zerovec");
         if self.is_empty() {
```

Comment:
Thought: we may want to be explicit about the type parameter in the generated code in order to be safer. Can do later.

Reply:
agreed